pwa/src/components/object_entities/dataForm.tsx

```diff
@@ -8,14 +8,19 @@ import {checkValues, removeEmptyObjectValues, retrieveFormArrayAsOArray} from ".
 import {navigate} from "gatsby-link";
 import ElementCreationNew from "../common/elementCreationNew"
 
-export default function ObjectEntityForm({id, entityId}) {
+interface ObjectEntityFormProps {
+  id: string
+  entityId
```

Comment:
Hier geldt hetzelfde, wat is de type van `entityId`?

In dit geval is het een string, dus: `entityId: string`.

Dit is belangrijk omdat (e.g.): nu zou ik een `Array` als `entityId` prop mee kunnen geven en dan krijg ik in de IDE geen error, maar dan gaat de applicatie wel kapot.

Reply:
Goed dat je het zegt niet aan gedacht